### ISSUE TITLE
MikroTik ROS support

### ISSUE THREAD

**Issue by MohammadSRamin** [donaldzou/WGDashboard#371]
hey so as you know in wireguard we dont have user managing and its so cool that you did it i was thinking about this for long time and i dont know python very much so i was going to write it with html php.
 but then in the middle of my project i saw your amazing work but it cannot manage ROS wireguard and i think this feature be so cool that make a web ui that can manage some ROS using even an ssh is soo cool if you can add this feature it will be soooo amazing for us mikrotik lovers.
 
 and if you needed help about ROS and commands of it need to run i can help you 
 
 p.s: i love your dashboard dude its amazed me when i saw it restricter the peer when he crossed over 1G data usage

**Comment by DaanSelen** [comment]
Hello @MohammadSRamin do you perhaps know what needs to be done to support this? I am not known with MikroTik?

**Comment by MohammadSRamin** [comment]
yes it don't need anything really mikrotik ROS has api service if you want to do it with api but i personally prefer ssh so let say the dashboard need a new section that get ssh credentials of mikrotik server/servers and using ssh it can give ssh commands and apply changes to mikrotik wiregruard the other things is the same but creating the wiregruard server and peers happens in the mikrotik server i think it would be so cool
and for the commands i can help you too if needed

**Comment by DaanSelen** [comment]
Sorry to say, but can you explain MikroTik in essence? I am mostly known with Microsoft Windows, Linux and Apple MacOS, so how does it work? Does it support Python, or just API/SSH?

**Comment by MohammadSRamin** [comment]
mikrotik itself is the company that work in networking and it has a operating system that is named RouterOS it do the tunneling vpning routing and other things like it but easier to do and in our use its wiregruard tunnel

**Comment by donaldzou** [comment]
Hi! This is kinda out of my knowledge scope and I don't have any hardware can run RouterOS... I will close this for now and hope I can figure this out in the future :)

**Comment by LucaDev** [comment]
I‘d love to help with this one! I have a lot of devices running RouterOS

**Comment by DaanSelen** [comment]
If you have time to make it work. Then by all means! And let us know how we can do it through a PR
